wiki/Tournaments/id.md

```diff
@@ -1,7 +1,3 @@
----
-outdated_translation: true
----
-
 # Turnamen
 
 Laman ini diperuntukkan untuk turnamen-turnamen yang ada di osu!. Turnamen memiliki forum khusus yang dapat kamu temukan [pada tautan berikut](https://osu.ppy.sh/community/forums/55).
```

Comment:
Maybe I suggest adding the following line to make it up-to-date.

```suggestion
Laman ini diperuntukkan untuk turnamen-turnamen yang ada di osu!. Turnamen memiliki forum khusus yang dapat kamu temukan [pada tautan berikut](https://osu.ppy.sh/community/forums/55).

Silahkan kunjungi halaman [dukungan resmi turnamen] untuk informasi mengenai cara mengajukan permintaan dukungan resmi untuk turnamenmu (lencana, *screening process*, dll.).
```

Reply:
sorry for my late reply but yup